subsys/mgmt/mcumgr/transport/Kconfig.bluetooth

```diff
@@ -87,4 +87,24 @@ config MCUMGR_SMP_BT_CONN_PARAM_CONTROL_RETRY_TIME
 
 endif # MCUMGR_SMP_BT_CONN_PARAM_CONTROL
 
+config MCUMGR_SMP_BT_AUTOMATIC_INIT
```

Comment:
Please change to `MCUMGR_TRASNPORT_<BT|UDP|UART>...` to have similar naming as proposed here https://github.com/zephyrproject-rtos/zephyr/pull/52761.

Reply:
Fixed